<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plankton ScriptCue A1.3.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="cueListLayout">
      <item>
       <widget class="QListWidget" name="cueListWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="cueListControls">
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prevButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addCueButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editCueButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeCueButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveCueUpButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="moveCueDownButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Move Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="quickButtonControls">
      <item>
       <widget class="QPushButton" name="editQuickButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Edit Quick Cues</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prevPageQuickButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Previous QC Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPageQuickButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Next QC Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>QC Page 1/1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="quickButtons">
      <item>
       <widget class="QPushButton" name="quickButton1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton8">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton9">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quickButton0">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
